src/lib/viewers/controls/settings/__tests__/Settings-test.tsx

```diff
@@ -93,5 +93,41 @@ describe('Settings', () => {
             expect(wrapper.exists(SettingsFlyout)).toBe(true);
             expect(wrapper.exists(SettingsToggle)).toBe(true);
         });
+
+        describe('disableTransitions prop', () => {
+            test('should default disableTransitions to false', () => {
+                const wrapper = getWrapper();
+
+                expect(wrapper.find(SettingsFlyout).prop('disableTransitions')).toBe(false);
+            });
+
+            test('should pass disableTransitions to along to SettingsFlyout', () => {
+                const wrapper = getWrapper({ disableTransitions: true });
+
+                expect(wrapper.find(SettingsFlyout).prop('disableTransitions')).toBe(true);
+            });
+        });
+
+        describe('icon prop', () => {
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            function CustomIcon({ isOpen, ...rest }: object, ref: React.Ref<HTMLDivElement>): JSX.Element {
```

Comment:
Do we need to destructure `isOpen` out here?

Reply:
It's to avoid a console error while running unit tests since `isOpen` isn't a valid attribute on a div